linkerd/protocol/http/src/main/scala/com/twitter/finagle/buoyant/linkerd/LinkerdHeaders.scala

```diff
@@ -49,6 +52,19 @@ import scala.collection.breakOut
 object Headers {
   val Prefix = "l5d-"
 
+  /**
+   * A param specific to error responses, the maximum size of a non-chunked HTTP error response payload.
```

Comment:
I don't think we ever chunk encode error responses.

Reply:
removed words from comment